### ISSUE TITLE
Facilitate alternative card database preferences

### ISSUE THREAD

**Issue by CajunAvenger** [Cockatrice/Cockatrice#4330]
**Similar Requests**
#4109 

**Description of New Feature**
Allow custom configurations of Cockatrice from startup, likely based on an ini file. This would change which cards/tokens xmls are loaded, the card source URLs, the download links for Oracle, and what those files are saved as (to consistently link up with the loaded xmls) automatically rather than needing all of those changed manually or separated into multiple portable installations. 

Ideally there'd be a way to set up shortcuts to start with particular inis rather than needing to choose every time on startup or edit file names or something.

**Additional Context**
This is from discussions with @ebbit1q on how to streamline custom usage of Cockatrice. They asked me to post this here as a place to work on it more.

**Comment by ebbit1q** [comment]
does it have to be on startup? wouldn't it be preferable to be able to switch while the application is running?

**Comment by CajunAvenger** [comment]
Sure, but its an inconvenience that happens exactly once, and not every few times they open Cockatrice.

> we should improve the current experience for all operating systems
and ye if the shortcut thing isn't doable everywhere then the setting thing is still good.

**Comment by olegshtch** [comment]
Is it something related to https://github.com/Cockatrice/Cockatrice/issues/4050 ?

**Comment by ebbit1q** [comment]
yes, similar to #1740 but specifically for the database instead of the player interface
